var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -51,6 +51,7 @@ class Paraview(Package):
     depends_on('qt@:4', when='+qt')
 
     depends_on('cmake', type='build')
+    depends_on('mesa')
```

Comment:
This is just to get it working for me (thus the WIP in PRs name) in the most hackish way, it's another question how to best incorporate it or what the right solution actually is. I don't know.

@tgamblin I build Qt via Spack. I first build Paraview without Qt, so there was no dependency and the build failed. Let me try to build Paraview with qt, it might actually work.